code/game/objects/items/weapons/cigs.dm

```diff
@@ -98,6 +98,13 @@ LIGHTERS ARE IN LIGHTERS.DM
 		if(L.lit)
 			light("<span class='notice'>After some fiddling, [user] manages to light [user.p_their()] [name] with [L].</span>")
 
+	else if(istype(I, /obj/item/match/unathi))
+		var/obj/item/match/unathi/U = I
+		if(U.lit)
+			playsound(U.loc, 'sound/effects/unathiignite.ogg', 40, 0)
```

Comment:
```suggestion
			playsound(U.loc, 'sound/effects/unathiignite.ogg', 40, FALSE)
```
Could also rename the sound to ```unathi_ignite``` to avoid that double "i"?

Reply:
unathihiss was named like that, I followed that. Can change it tho.